### ISSUE TITLE
Lost 6 soul points by dying once

### ISSUE THREAD

**Issue by ecwarriorz** [Wynncraft/Issues#4039]
It's just the lag, right?
![screen shot 2016-02-07 at 11 37 26 pm](https://cloud.githubusercontent.com/assets/16756977/12883195/4ede0c3e-ce91-11e5-91f7-838e479760ea.png)

**Comment by Moe-Ronickah** [comment]
Can you give us any more details on this?  Where were you when you died?  Where you AFK at the time?  Did you stop for a few moments to go do something else, then come back and find this on your screen?  Any other information that you have would be really appreciated.

**Comment by DarkDemonDragon** [comment]
@Moe-Ronickah just a side question what sort of position are you on Github? An official staff or just volunteer.

**Comment by TrashcanMan** [comment]
@DarkDemonDragon I assume he's  staff since he's able to add tags, close issues, etc.

**Comment by Dar-krusos** [comment]
@DarkDemonDragon He even got admin powers on the wiki. Ha

**Comment by Moe-Ronickah** [comment]
Assistant admin is what I would call it.  I don't touch the coding for the GitHub, but I have full admin powers, the same as Trixter.

**Comment by Dar-krusos** [comment]
Man, I can't take the Wynncraft admins seriously.

**Comment by ecwarriorz** [comment]
I was in the Ice Mini-Dungeon at the edge of Nesaak Forest when I died. I was trying to fight off a bunch of mobs, and I spammed my spells right before dying. After dying, I ended up back in Nesaak with a lot less soul points than expected.

**Comment by Dar-krusos** [comment]
Yeah, this shouldn't be happening. Did you go AFK after you died? If so, mobs are able to kill you, just like in Lexdale.

**Comment by DarkDemonDragon** [comment]
@ecwarriorz did the exact same thing happen with chat? Did it say:
"You have died...
2 Soul points have been lost"
more than once at the same time?

**Comment by ecwarriorz** [comment]
@DarkDemonDragon Yes. This message appeared 3 times in the span of 1 second. I wasn't AFK.

**Comment by Trixter313** [comment]
Duplicate of https://github.com/Wynncraft/Issues/issues/3821

Please direct any future comments to the original thread. Thanks!

**Comment by DarkDemonDragon** [comment]
@Trixter313 this is not a duplicate. This issue is when players lose more soul points than normally. That issue is when players lose items when not at 0 SP.

**Comment by RussellTheEndbringer** [comment]
This happened to me also. What is going to happen with this?

**Comment by Moe-Ronickah** [comment]
@RussellTheEndbringer , because the issue is still alive and well, unfortunately, it's still being looked at to try to find out what is  causing it.

**Comment by RussellTheEndbringer** [comment]
Do you know what will happen to the players' items?

**Comment by Moe-Ronickah** [comment]
If the player dies, the items get dropped.  Anyone can pick them up after 20seconds.

**Comment by RussellTheEndbringer** [comment]
I mean, I lost my stuff that I worked hard for. Will our inventories get rolled back, or will there be a form of compensation for our losses?

**Comment by Trixter313** [comment]
Unfortunately Wynncraft has a strict no refunds policy so most likely there's nothing that can be done about people's lost items. Every player's inventory would have to be rolled back to the date that someone first experienced the issue.

**Comment by RussellTheEndbringer** [comment]
I have the strangest feeling that someone just reported Wynncraft to EULA for it's multiple violations.

**Comment by YunoFGasai** [comment]
@RussellTheEndbringer wynncraft doesn't violate the Eula and using it as a threat wont speed up the process, getting a rollback means everyone gets a rollback which only happens if a bug effected a majority of the player base.

**Comment by RussellTheEndbringer** [comment]
A big group of my friends got together and red the EULA and then made joint decisions about what to put in the report. The only involvement I had was telling them that I really didn't care what they did, so don't shoot the messenger. But yes, Wynn does violate EULA if you read it carefully.

**Comment by Moe-Ronickah** [comment]
**This Issue is about the loss of an excessive number of Soul Points under unknown conditions.  Please remain on topic.**

**Comment by YunoFGasai** [comment]
@RussellTheEndbringer let's continue this on the forums.

**Comment by DebugingLonks** [comment]
Not trying to say anything here other than I have had this once. However i lost 4 not 2. I wouldn't try to re-create it tho... as you loose sp xD

Basicly I was being attacked by a ton of mobs (Just like this guy) and I got killed. Its possible it just magicly killed us same time.

**Comment by ecwarriorz** [comment]
Honestly, I think it's a one-time thing that happens because of lag. I don't think I can ever get it to happen again.

**Comment by RussellTheEndbringer** [comment]
All fine and whatnot when it isn't you.


Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Erela <notifications@github.com> Date: 1/3/17  8:10 PM  (GMT-05:00) To: Wynncraft/Issues <Issues@noreply.github.com> Cc: RussellTheEndbringer <russelljlea@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [Wynncraft/Issues] Lost 6 soul points by dying once (#4039) 
Honestly, I think it's a one-time thing that happens because of lag. I don't think I can ever get it to happen again.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Wynncraft/Issues","title":"Wynncraft/Issues","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Wynncraft/Issues"}},"updates":{"snippets":[{"icon":"PERSON","message":"@ecwarriorz in #4039: Honestly, I think it's a one-time thing that happens because of lag. I don't think I can ever get it to happen again. "}],"action":{"name":"View Issue","url":"https://github.com/Wynncraft/Issues/issues/4039#issuecomment-270270730"}}}

**Comment by ecwarriorz** [comment]
Well, it was me. I started the thread.

**Comment by DebugingLonks** [comment]
^

**Comment by RussellTheEndbringer** [comment]
Well what are we going to do about it?

**Comment by RussellTheEndbringer** [comment]
...
